### PR TITLE
Override custom header colors with ITB colors

### PR DESCRIPTION
**Pull request by brianhlin**
Worked for `docs-itb` here: https://travis-ci.org/opensciencegrid/docs/builds/425434247

### CODE REVIEW

File: deploy [hunk 1]
```diff
@@ -25,7 +25,10 @@ elif [[ "${TRAVIS_BRANCH}" =~ $ITB_BRANCH_REGEXP ]]; then
     sed -i 's/\(extra_css:\)/\1\n  - css\/itb.css/' mkdocs.yml
     CSS_DIR='docs/css/'
     mkdir -p $CSS_DIR
-    cp ci/itb.css $CSS_DIR
+    # If we set the colors in extra.css, replace them with the ITB colors.
+    # Otherwise, copy over the ITB color CSS.
+    sed -i 's/#343a40/#CC0000/' $CSS_DIR/extra.css || cp ci/itb.css $CSS_DIR
```
Comment (edquist) [inline deploy]: Pardon my ignorance, but where does `extra.css` come from?  Did you perhaps mean `itb.css` ?

Comment (brianhlin) [reply deploy]: [extra.css](https://github.com/search?q=org%3Aopensciencegrid+extra.css&type=Code) is used in many (all?) of the mkdocs repos

Comment (edquist) [reply deploy]: Ok, i'm not sure i understand what is happening with the `sed ... || cp ...` logic then.

What is the failure condition for `sed` that you want to trigger the `|| cp ...` part?

(Unlike `grep`, `sed` returns success even when no lines are matched/replaced - i would expect it to fail there only if `extra.css` does not exist, or if there is some other io error.)

Comment (brianhlin) [reply deploy]: D'oh, I knew that but didn't really think it through. I was just trying to catch the case where `extra.css` didn't exist with the above. 

But if `sed` fails to replace any lines, we still want to copy `itb.css`.

File: deploy [hunk 2]
```diff
@@ -27,8 +27,9 @@ elif [[ "${TRAVIS_BRANCH}" =~ $ITB_BRANCH_REGEXP ]]; then
     mkdir -p $CSS_DIR
     # If we set the colors in extra.css, replace them with the ITB colors.
     # Otherwise, copy over the ITB color CSS.
-    sed -i 's/#343a40/#CC0000/' $CSS_DIR/extra.css || cp ci/itb.css $CSS_DIR
+    sed -i 's/#343a40/#CC0000/' $CSS_DIR/extra.css ||:
```
Comment (edquist) [inline deploy]: This is better, but to me the `||:` things still seem a bit odd.  Their effect is, "`set -e` is in effect, but nevertheless don't exit the script early if these `sed` commands error out when, eg, `extra.css` does not exist."

Is that what you want?  That is, `extra.css` being missing is OK?

Comment (brianhlin) [reply deploy]: Yeah, it's fine if it doesn't exist, copying in `itb.css` ensures that we have the right config in place.

That being said, maybe I should just wrap the seds in `set +e...set -e`

Comment (edquist) [reply deploy]: Nah it's fine to use `|| :` as long as that's the semantics you want.

**Review by edquist — APPROVED** [review]
`||:`'s made me O_o a bit but Brian says that's really what he means.  Alright then!

**Pull request merged**